packages/paste-core/components/combobox/src/ComboboxListboxGroup.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import {Box} from '@twilio-paste/box';
+import {Text} from '@twilio-paste/text';
+
+export interface ComboboxListboxGroupProps {
+  children: NonNullable<React.ReactNode>;
+  groupName: string;
+}
+
+const ComboboxListboxGroup: React.FC<ComboboxListboxGroupProps> = ({children, groupName}) => {
+  return (
+    <Box as="div" role="group" aria-label={groupName} paddingTop="space20">
```

Comment:
Is `role="group"` working as intended here? Just double checking if it was just a `Menu` issue

Reply:
Works a treat 👍